src/styles/utils/cards.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+
+  &__image {
+    align-self: center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: inherit;
+    line-height: 18px;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 23px;
+
+    &-tag {
+      font-size: 10px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-tag {
+      font-size: inherit;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__button {
+    font-weight: 700;
+    font-size: 14px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: white;
+    background-color: $color-accent-blue;
+    text-decoration: none;
+    text-transform: uppercase;
+    border: 1px solid $color-accent-blue;
+    border-radius: 5px;
+    transition: color, background-color 0.3s ease-in-out;
+
+    &:hover {
+      background-color: white;
+      color: $color-accent-blue;
+      border: 1px solid $color-accent-blue;
```

Comment:
You can add a border for the default state of card and change only border-color on hover